### DISCUSSION TITLE
表单设计器可以离线运行吗

### DISCUSSION THREAD

**Discussion by diaomina** [alibaba/formily#2010] [Q&A 2.x]
1、表单设计器可以离线运行吗，看到里面请求了4个js后下载下来发现js内还有请求，控制台报错。
2、是否可以通过组件的形式嵌入其他系统
![image](https://user-images.githubusercontent.com/40923075/129440450-4a146229-d0e2-403b-be5d-0d872f0f06be.png)

![image](https://user-images.githubusercontent.com/40923075/129440415-a177e4e5-374a-43b9-9b8d-129e2a2b925c.png)

**Comment by fuzi1996** [comment]
参见 => https://github.com/alibaba/formily/discussions/1988
